### PR TITLE
doc/stdenv: document default build scripts filename

### PR DESCRIPTION
**Pull request by tbenst**
explain where default scripts come from

### CODE REVIEW

File: doc/stdenv.xml [hunk 1]
```diff
@@ -747,7 +747,8 @@ passthru.updateScript = [ ../../update.sh pname "--requested-release=unstable" ]
    latter is convenient from a build script. However, typically one only wants
    to <emphasis>add</emphasis> some commands to a phase, e.g. by defining
    <literal>postInstall</literal> or <literal>preFixup</literal>, as skipping
-   some of the default actions may have unexpected consequences.
+   some of the default actions may have unexpected consequences. The default
+   script for each phase is defined in the package stdenv/setup.
```
Comment (vcunat) [inline doc/stdenv.xml]: I'm not sure if that's really clear.  Perhaps "file `pkgs/stdenv/generic/setup.sh`"?

**Comment by etu** [comment]
The commit message is very unclear, other than a file is changed. Nothing about why or so. Clarification on what this commit does and the reasoning behind it would be great as well.

**Comment by worldofpeace** [comment]
Please prefix the commit message with `doc/stdenv` as well. (see git history)

**Comment by tbenst** [comment]
@etu, @worldofpeace thanks! Done & will keep in mind for future pull requests

**Comment by worldofpeace** [comment]
You're welcome @tbenst. And thank you for contributing :sparkles:

**Pull request merged**